miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/LibraryDilutionRestController.java

```diff
@@ -83,15 +85,15 @@ public void setDilutionService(LibraryDilutionService dilutionService) {
     this.dilutionService = dilutionService;
   }
 
-  @RequestMapping(value = "{dilutionId}", method = RequestMethod.GET, produces = "application/json")
+  @GetMapping(value = "{dilutionId}", produces = "application/json")
   @ResponseBody
   public DilutionDto getDilution(@PathVariable Long dilutionId) throws IOException {
     LibraryDilution dilution = dilutionService.get(dilutionId);
     DilutionDto dilutionDto = Dtos.asMinimalDto(dilution);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "dilutionDto". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)